api/routes.go

```diff
@@ -41,32 +41,32 @@ func (api *API) buildHTTPRoutes() {
 
 	api.staticRouter.GET("/login", api.WithDBSession(api.noAuth(api.loginGET)))
```

Comment:
still needs a session?

Reply:
so I guess for these routes the reason you still need a session is because of the challenge right?
just wondering whether it's an issue you have more than one challenge?

Reply:
There is a theoretical situation in which we delete the user and then try to create a challenge for them. That's what makes the session necessary. Not that it's going to have much of an effect, aside from leaving junk in the DB.

Reply:
ok, this also means a login might block a delete though right, feel free to resolve it was just a question